src/Orc.NuGetExplorer/Models/NugetFeed.cs

```diff
@@ -119,17 +124,24 @@ private void ValidateAndRaiseErrorsChanged(string propertyName)
         {
             var error = this[propertyName];
 
-            if (!_propertyNameToDataError.TryGetValue(propertyName, out string oldError))
+            if (!_propertyNameToDataError.TryGetValue(propertyName, out var oldError))
             {
                 oldError = string.Empty;
             }
 
-            if (!string.IsNullOrEmpty(error))
+            if (string.IsNullOrEmpty(error))
+            {
+                if (_propertyNameToDataError.ContainsKey(propertyName))
```

Comment:
I think there is no need to check for existence. I think REmove returns a bool whether it was a success